ros_buildfarm/config/index.py

```diff
@@ -87,6 +88,13 @@ def __init__(self, data, base_url):
                 for key in unset_keys:
                     self.distributions[distro_name][key] = value_types[key]()
 
+        self.ci_builds = {}
+        if 'ci_builds' in data and data['ci_builds']:
+            assert isinstance(data['ci_builds'], dict)
+            for k, v in data['ci_builds'].items():
+                v = _resolve_url(base_url, v)
+                self.doc_builds[k] = v
```

Comment:
Is this a copy paste error? Should it be?

```suggestion
                self.ci_builds[k] = v
```